### PR TITLE
autotools plugin: support bootstrap parameters

### PR DESCRIPTION
**Pull request by sergiusens**
- [ ] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?

-----

### CODE REVIEW

File: craft_parts/plugins/autotools_plugin.py [hunk 1]
```diff
@@ -28,6 +28,7 @@ class AutotoolsPluginProperties(PluginProperties, PluginModel):
     """The part properties used by the autotools plugin."""
 
     autotools_configure_parameters: List[str] = []
+    autotools_bootstrap_parameters: List[str] = []
```
Comment (tigarmo) [inline craft_parts/plugins/autotools_plugin.py:31]: Add this to `AutotoolsPlugin`'s docstring please

Comment (sergiusens) [reply craft_parts/plugins/autotools_plugin.py:31]: done

**Review by cmatsuoka — APPROVED** [review]
Looks good! But a test is failing.

File: tests/integration/plugins/test_autotools.py [hunk 2]
```diff
@@ -1 +1,64 @@
...
+        work_dir=new_dir,
+        partitions=partitions,
+    )
+    actions = lf.plan(Step.PRIME)
+
+    with lf.action_executor() as ctx:
+        ctx.execute(actions)
+
+    prime_dir = lf.project_info.prime_dir
+    hello_binary = Path(prime_dir, "bin", "hello")
```
Comment (cmatsuoka) [inline tests/integration/plugins/test_autotools.py:60]: The configuration prefix is `/usr`, this should be reflected here.

**Pull request merged**